server/server.go

```diff
@@ -246,6 +252,7 @@ func (s *Server) indexHandler(w http.ResponseWriter, r *http.Request) {
 		OpenshiftConsoleURL: s.OpenshiftConsoleURL,
 		LogoImageName:       s.LogoImageName,
 		GoogleTagManagerID:  s.GoogleTagManagerID,
+		LoadTestFactor:      s.LoadTestFactor,
```

Comment:
And this as well, what was wrong with getting this as a query parameter? If this is just being used for dev testing it might make sense to just have this passed in as a query parameter and not having the requirement to restart the server.

Reply:
Using query parameter means you have to refresh the page twice: once to `postMessage` the Service Worker to tell it what factor to use, then again for the Service Worker to intercept and duplicate the responses. Also restarting the server is fast, and the command-line flag is better documented as part of the app instead of in the repo somewhere.